csrc/device_lower/utils.cpp

```diff
@@ -783,8 +785,8 @@ BasicAllocInfo getAllocInformation(
       }
     }
 
-    if (GpuLower::current()->caMap()->areMapped(
-            local_id, fl_id, IdMappingMode::PERMISSIVE)) {
+    if (lower_utils::getConcreteLoopID(local_id) ==
```

Comment:
This part needs to be changed as we are no longer using the CA map if the option is enabled.